src/object_finding/feature_comparator.cpp

```diff
@@ -0,0 +1,93 @@
+#include "feature_comparator.hpp"
+
+namespace lucidy{
+
+    FeatureComparator::FeatureComparator(lucidy::settings::OBF::data & settings):
+        settings(settings),
+        detector(settings.fdt),
+        descriptor(detector, settings.fdc)
+    {initMatcher();}
```

Comment:
bit of a weird construction with the initMatcher() function...
Place whatever should be in initMatcher in the constructor instead

Reply:
This makes code unnecessarily long and init function is alway more recommend. Invalid comment due to bad advice